Packages/com.unity.inputsystem/InputSystem/Events/InputEventBuffer.cs

```diff
@@ -59,7 +59,7 @@ public long capacityInBytes
                 if (!m_Buffer.IsCreated || m_BufferEnd == kBufferSizeUnknown)
                     return 0;
 
-                return m_BufferEnd - m_Buffer.Length;
+                return m_Buffer.Length - m_BufferEnd;
```

Comment:
I cant figure out how to get a larger diff, is this a correct change? nothing else in this file changed, so im assuming this fixes a bug somewhere.

Reply:
This is returning the amount available in a semi-used buffer.  
BufferEnd is the current in-use amount, and Buffer.Length is the entire container.
This means that buffer end is always smaller then (or equal to) the total potential space, so it would result in a negative capacity.

We want the gap between the total capacity and the current end of the buffer.